copier/main.py

```diff
@@ -766,7 +792,7 @@ def run_recopy(self) -> None:
                 f"from `{self.subproject.answers_relpath}`."
             )
         with replace(self, src_path=self.subproject.template.url) as new_worker:
-            return new_worker.run_copy()
+            new_worker.run_copy()
```

Comment:
If `new_worker.run_copy()` returns something, that would be a breaking change, even if we didn't advertise the return value because of our `None` return annotation. Need to decide whether we're OK with the breaking change or if we should update the return annotation to reflect the actual return value.

Reply:
i'm not sure what you mean?
`new_worker.run_copy()` doesn't return anything, and there's no return statement in that function anyway

Reply:
Great, so the condition in my comment is never true, and we can discard the whole comment :slightly_smiling_face: